src/Faker/Calculator/Iban.php

```diff
@@ -17,24 +17,20 @@ public static function checksum($iban)
         $checkString = substr($iban, 4) . substr($iban, 0, 2) . '00';
 
         // Replace all letters with their number equivalents
-        $checkString = preg_replace_callback('/[A-Z]/', ['self', 'alphaToNumberCallback'], $checkString);
+        $checkString = preg_replace_callback(
+            '/[A-Z]/',
+            static function (array $matches): string {
+                return (string) self::alphaToNumber($matches[0]);
+            },
+            $checkString,
+        );
 
         // Perform mod 97 and subtract from 98
         $checksum = 98 - self::mod97($checkString);
 
         return str_pad($checksum, 2, '0', STR_PAD_LEFT);
     }
 
-    /**
-     * @param string $match
-     *
-     * @return int
-     */
-    private static function alphaToNumberCallback($match)
```

Comment:
Running

```shell
git grep -i alphaToNumberCallback
```

yields nothing in this branch, so I believe we can safely remove it.